test/verify/check-metrics

```diff
@@ -793,6 +793,11 @@ class TestCurrentMetrics(MachineCase):
         with b.wait_timeout(300):
             b.wait_in_text("#current-metrics-card-cpu", f"pod {shortid}")
```

Comment:
This is now a race condition. It could be that the name lookup is fast enough that we don't see the short ID on the page ever. But *shrug*, it seems to work in CI, so fine for now. Just something to watch out for later.